picky/Cargo.toml

```diff
@@ -34,6 +34,7 @@ serde_json = { version = "1.0.93", optional = true }
 http = { version = "0.2.9", optional = true }
 cab = { version = "0.3.0", optional = true }
 lexical-sort = { version = "0.3.1", optional = true }
+zeroize = "1.5"
```

Comment:
Well, we were not zeroing EC private keys for `EcdsaKeypair` previously (was here for quite some time now) and I haven't noticed it previously too when extending support of EC in my previous PR, so `PrivateKeyKind::Ec` added by me was not zeroing either... but now I spotted it and fixed this and added zeroing logic to `EcdsaKeypair`, `EdKeypair` and `PrivateKeyKind`, so now we should be a little bit more secure

Reply:
Sounds good